video/out/drm_common.c

```diff
@@ -1240,6 +1281,43 @@ double vo_drm_get_display_fps(struct vo_drm_state *drm)
     return mode_get_Hz(&drm->mode.mode);
 }
 
+bool vo_drm_set_hdr_metadata(struct vo *vo)
+{
+    struct vo_drm_state *drm = vo->drm;
+    struct mp_image_params target_params = vo_get_target_params(vo);
+    if (!vo->target_params || pl_color_space_equal(&target_params.color, &drm->target_params.color))
+        return false;
+
+    destroy_hdr_blob(drm);
+    drm->target_params = target_params;
+
+    struct pl_hdr_metadata hdr = target_params.color.hdr;
```

Comment:
```suggestion
    const struct pl_hdr_metadata *hdr = &target_params.color.hdr;
```
no need to copy.

Also we should check if colorspace is hdr before sending it, I think. 

Also 
> Userspace first need to detect the HDR capabilities of sink by reading and parsing the EDID.

I don't think we do that? Probably nothing terrible would happen when we send blob to unsupported display, but what if?

Reply:
We do need to send it in the sdr case so the display can unset itself if the playlist switches from an hdr file to an sdr one. It is not like we are sending anything wrong so it should be OK.

>Probably nothing terrible would happen when we send blob to unsupported display, but what if?

Well just nothing happens but it's probably worth checking so I'll add that.